app/alert_screen.py

```diff
@@ -0,0 +1,256 @@
+                            },
+                        ),
+                        dbc.Row(
+                            id="fire_text_row",
+                            children=fire_text_div,
+                            style={
+                                "display": "flex",
+                                "justify-content": "center",
+                                "margin-right": "2.5%",
+                            },
+                        ),
+                    ],
+                    style={
+                        "width": "50%",
+                        "margin": "2.5%",
+                    },
+                ),
+                dbc.Col(
+                    id="col_image_fire",
+                    children=[
+                        dbc.Row(
+                            html.Div(
+                                html.Div(
+                                    html.P("DÉPART DE FEU"),
+                                    style={
+                                        "font-size": "4vw",
+                                        "color": "#fd4848",
+                                        "font-weight": "bold",
+                                    },
+                                    className="blink-image",
+                                ),
+                            ),
+                            style={
+                                "display": "flex",
+                                "justify-content": "center",
+                                "padding-bottom": "7%",
+                            },
+                        ),
+                        dbc.Row(
+                            id="fire_images_row",
+                            children=fire_images_div,
+                            style={
+                                "display": "flex",
+                                "justify-content": "center",
+                                "margin-right": "2.5%",
+                            },
+                        ),
+                    ],
+                    style={
+                        "width": "50%",
+                        "margin": "2.5%",
+                    },
+                ),
+            ],
+            style={
+                "height": "100%",
+            },
+        ),
+    ]
+
+    style = {
+        "height": "100%",
+        "width": "100%",
+        "position": "fixed",
+    }
+
+    return layout_div, style
+
+
+# ----------------------------------------------------------------------------------------------------------------------
+# App layout
+# The following block gathers elements defined above and returns them via the alert_screen function
+def alert_screen():
+    """
+    The following function is used in the main.py file to build the layout of the big screen page.
+    """
+    layout = html.Div(
+        children=[
+            dcc.Interval(
+                id="interval-component-alert-screen",
+                interval=3 * 1000
```

Comment:
I'm not sure to understand why there are 2 `dcc.Interval ` here and inside `fire_images_div`, the `interval-component-img-refresh` would not be enough for the gif ?

Reply:
The Interval `interval-component-homepage` is used to call the API and update the alerts metadata (see function `update_alert_data()`). I could not use it to synchronize the callback `update_alert_screen()`: I had the error `a nonexistent object was used in an output of a dash callback` due to the fact that the div `core_layout_alert_screen` does not exist in the `homepage` (precisely because it is not meant to be on this page).

This is why I had to create the Interval `interval-component-alert-screen`: in order to update the alert page display (make the switch between the alert screen and the no alert screen when needed). 

And yes the Interval `interval-component-img-refresh` is used for the GIF.